### PR TITLE
DANS Archiving, Bag, and OREMap updates

### PR DESCRIPTION
**Pull request by qqmyers**
**What this PR does / why we need it**: Adds some additional information to the OAI_ORE map file included in the Archival Bags (and used as the default OAI_ORE metadata export) and defines an official v1.0.0 for the format of this file (after which further changes should require updating the format version). Initial changes in this draft PR:
Adding the OAI_ORE version info to the map file
Adding info about the software/version of Dataverse used to create the Bag/OAI_ORE file
Adding info about the version state (draft, released, deaccessioned, archived) to the OAI_ORE including any deaccession reason and forwarding url provided.

**Which issue(s) this PR closes**:

Closes #

**Special notes for your reviewer**:

**Suggestions on how to test this**:

**Does this PR introduce a user interface change? If mockups are available, please link/include them here**:

**Is there a release notes update needed for this change?**:

**Additional documentation**:

### CODE REVIEW

**Comment by scolapasta** [comment]
@qqmyers  I'll move this milestone 6.1 issue to "ready for Review", but I'm assigning you first so that you can handle the 6.0 merge and address any EE10 issues.

**Comment by sekmiller** [comment]
Hey @qqmyers - after you updated from dev and took yourself off the ticket, did you intend to leave the PR in draft? (is there anything else you need to do here?)
Also, is there a change ticket that will be closed by the PR?
Thanks!

**Comment by qqmyers** [comment]
Sorry - probably should have left myself on it. The code is reviewable at this point but I need to add docs/release note. There isn't a corresponding issue in this repo (there is one at DANS).

File: src/main/java/edu/harvard/iq/dataverse/util/bagit/OREMap.java [hunk 1]
```diff
@@ -283,7 +327,11 @@ public JsonObjectBuilder getOREMapBuilder(boolean aggregationOnly) {
                             aggBuilder.add(JsonLDTerm.ore("aggregates").getLabel(), aggResArrayBuilder.build())
                                     .add(JsonLDTerm.schemaOrg("hasPart").getLabel(), fileArray.build()).build())
                     // and finally add the context
-                    .add("@context", contextBuilder.build());
+                    .add("@context", contextBuilder.build())
+                    ;
+
```
Comment (sekmiller) [inline src/main/java/edu/harvard/iq/dataverse/util/bagit/OREMap.java:332]: Extra blank lines misplaced ';' here?

**Review by sekmiller — CHANGES_REQUESTED** [review]
Looks fine except for a minor typo/formatting issue

File: src/main/java/edu/harvard/iq/dataverse/util/bagit/OREMap.java (hunk 1 above)
Comment (qqmyers) [reply src/main/java/edu/harvard/iq/dataverse/util/bagit/OREMap.java]: thanks - fixed

**Review by sekmiller — APPROVED** [review]

**Pull request merged**